plutus-core/executables/Common.hs

```diff
@@ -302,10 +417,10 @@ printmode = option auto
         ++ "Readable -> prettyPlcReadableDef, ReadableDebug -> prettyPlcReadableDebug" ))
 
 printOpts :: Parser PrintOptions
-printOpts = PrintOptions <$> languagemode <*> input <*> printmode
+printOpts = PrintOptions <$> input <*> printmode
```

Comment:
I wonder if we should change the default printing mode to `Debug` above?  If you convert from, say, `flat` to textual plc then all the variables are printed as `v` in `Classic` mode and that keeps catching people out (me included).  With `Debug` it adds the uniques so that you get names like `v_4241` and then you can parse it again without problems. We had a discussion about changing the `Pretty` instances because of this recently but unfortunately I can't find that.   

Anyway, I think changing the default would probably be a big help with this problem.  @michaelpj, @effectfully ?

Reply:
I'm all for using the `Debug` mode by default.

Reply:
Done, I think.